python/cudf/cudf/_lib/pylibcudf/libcudf/lists/contains.pxd

```diff
@@ -12,17 +13,33 @@ from cudf._lib.pylibcudf.libcudf.scalar.scalar cimport scalar
 
 
 cdef extern from "cudf/lists/contains.hpp" namespace "cudf::lists" nogil:
+
+    cpdef enum class duplicate_find_option(int32_t):
+        FIND_FIRST "cudf::lists::duplicate_find_option::FIND_FIRST"
+        FIND_LAST "cudf::lists::duplicate_find_option::FIND_LAST"
```

Comment:
The extra quoted parts were necessary when we did `ctypedef enum...`, but it will work automatically without that now that we're using a `cpdef enum class`.
```suggestion
        FIND_FIRST
        FIND_LAST
```